google/cloud/spanner/client.h

```diff
@@ -326,14 +326,16 @@ class Client {
   StatusOr<std::int64_t> ExecutePartitionedDml(SqlStatement const& statement);
 
   /**
-   * Commits a transaction.
+   * Commits a read-write transaction.
    *
    * The commit might return an `ABORTED` error. This can occur at any time;
    * commonly, the cause is conflicts with concurrent transactions. However, it
    * can also happen for a variety of other reasons. If `Commit` returns
    * `ABORTED`, the caller should re-attempt the transaction from the beginning,
    * re-using the same session.
    *
+   * @warning Read-only transactions are not permitted to call `Commit`.
```

Comment:
The wording sounds a little funny to my ear, since Transactions don't call anything. Maybe "It is an error to call `Commit` with a read-only `Transaction`."?